vendor/github.com/openshift/source-to-image/pkg/build/strategies/sti/sti.go

```diff
@@ -562,7 +562,6 @@ func (builder *STI) Execute(command string, user string, config *api.Config) err
 		}()
 
 		opts.Stdin = r
-		defer wg.Wait()
```

Comment:
not worth redoing the PR in s2i and then updating this PR for that change though.

we can make the change in s2i and pick it up on the next actual s2i bump if it's deemed that crucial to make the change.  All the return paths are currently covered by the existing logic.